### ISSUE TITLE
Script version, how to check?

### ISSUE THREAD

**Issue by mcfrojd** [Telekatz/venus.dbus-shellyPlug#17]
Is there a way to see what version of your script that is installed on a system by looking at the files in /data/dbus-shellyPlug ?
Or is there any other way?

**Comment by Telekatz** [comment]
No, there is no version number inside the code. However, one possibility would be to check the creation date of the files in the directory and determine which version was the current version at that time.
